src/wordpress/backup.py

```diff
@@ -90,7 +89,7 @@ def get_daily_list(self):
         # build regex for filenames with found dates
         matches = "|".join([valid_date.strftime("%Y%m%d")
                             for valid_date in valid_dates])
-        file_regex = re.compile("({})\d+.list".format(matches))
+        file_regex = re.compile("\d+.list".format(matches))
```

Comment:
Pourquoi tu vires ça ? sauf si je comprends pas, t'en as toujours besoin pour identifier les X derniers backups.

Reply:
Punaise, excellent commentaire. J'ai cru que cette partie servait uniquement à retrouver tout ce qui n'était pas décimal, et vu que j'ai viré le nom du laboratoire dans le fichier (voir la description dans la PR). J'ai mon regex interne qui s'est trompé sur ce coup là. Merci Lucien !
D'ailleurs, ne faudrait-il pas laisser le nom du site dans le nom du fichier de sauvegarde ? Qu'en penses-tu ? Cela peut être utile si on copie colle directement le fichier au lieu du répertoire. Et peut-être ajouter aussi le nom du site **et** le nom de l'environnement

Reply:
pour info sur la suppression des paranthèses, voici mon code de test :

`
import re
import datetime

filename_in = "20181016153303_full.sql"
filename_not_in = "20181016153303_full.sql"

valid_dates = [datetime.datetime.now() - datetime.timedelta(days=delta) for delta in range(25)]
matches = "|".join([valid_date.strftime("%Y%m%d") for valid_date in valid_dates])

file_regex_with_bracket = re.compile("({})\d+.list".format(matches))
file_regex_without_bracket = re.compile("{}\d+.list".format(matches))

print file_regex_with_bracket.search(filename_in)
> None
print file_regex_without_bracket.search(filename_in)
> <_sre.SRE_Match object at 0x7feb29a28718>

print file_regex_with_bracket.search(filename_not_in)
> None
print file_regex_without_bracket.search(filename_not_in)
> None
`